01-status.md

```diff
@@ -1,25 +1,25 @@
-## Status of this document
+# Status of this document
```

Comment:
only changing the level of this part of the specification will make that section the root of all the others, which isn't what we expect. If you want to make this change, you'll have to modify all the titles in the others files.

Reply:
Ah silly me, I forgot how this would appear on spec.md as a whole.. Reverting now!